CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 3.2.0
+* CHANGE: Callbacks set via `setDataCallback`, `setShouldSendCallback` now receive any prior-set callback as the 2nd argument. See: https://github.com/getsentry/raven-js/pull/636
+* CHANGE: Raven.js no longer passes a 'message' interface for exceptions. See: https://github.com/getsentry/raven-js/pull/632
```

Comment:
I'm not entirely sure off hand, that's just why I raised the question. It may be completely fine, I just don't know without investigating.